.github/workflows/someip2val_build.yml

```diff
@@ -150,3 +151,6 @@ jobs:
           tags: |
             ttl.sh/kuksa.val.feeders/someip-feeder-${{github.sha}}:8h
           labels: ${{ steps.meta.outputs.labels }}
+          # Provenance to solve that an unknown/unkown image is shown on ghcr.io
+          # Same problem as described in https://github.com/orgs/community/discussions/45969
+          provenance: false
```

Comment:
@int0x27 - It turns out the changed buildx-version caused by the update to build-push-action@v4 two months ago caused oci-manifests to be used, rather than docker manifests. That can be solved by adding `provenance: false`.

This only seems to affect someip-feeder, as the others use our own builder and thus does not use `setup-buildx-action`.


Before adding `provenance: false`

```
erik@debian3:~/kuksa.val/proto/kuksa/val/v1$ docker manifest inspect ttl.sh/kuksa.val.feeders/someip-feeder-44e43a1defc75444feaefd60d5480accbe06309a:8h
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.oci.image.index.v1+json",
   "manifests": [
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 1620,
         "digest": "sha256:7e1c4745d5d7d279cfa94318cf453e548656d897a6d63c9cafec003f774ca7ab",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 1620,
         "digest": "sha256:410250a82b58bd5e1485762f90e70cea89ae7d203a89deff22f292de10492c2f",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 567,
         "digest": "sha256:4d89de90d5b49718be1184231b8bc715f14a53b4f50a67b30c0ac912aea5ea68",
         "platform": {
            "architecture": "unknown",
            "os": "unknown"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 567,
         "digest": "sha256:b511431bdd319f11785cb08eaf89a51d5f66595de4d7bd456c51165480a74c23",
         "platform": {
            "architecture": "unknown",
            "os": "unknown"
         }
      }
   ]
}
```
After adding `provenance: false`

```
erik@debian3:~/kuksa.val/proto/kuksa/val/v1$ docker manifest inspect ttl.sh/kuksa.val.feeders/someip-feeder-e274e81775597a3c11357a3ee40e3e809ce4b4f8:8h
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1678,
         "digest": "sha256:df84e70de39310b04c1e27fcf8b21ae6f0ae6b3b035ed7da798a66c17125d564",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1678,
         "digest": "sha256:6b05b6e681892a9b09a16a4d2dbb46e1618ae00f82874e315b04bf96a28f6b78",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
erik@debian3:~/kuksa.val/proto/kuksa/val/v1$ 
```